syntax_and_semantics/type_workflow/strings.md

```diff
@@ -0,0 +1,26 @@
+# Working with Strings
+
+```crystal
+"abcd" # => "abcd" : String
+
+# Access a part of a string
+"abcd"[1..2] # => "bc" : String
+
+# Start at a position and continue for X characters
+"abcd"[2,2] # => "cd" : String
+
+# Concatenate strings
+"abcd" + "e" # => "abcde"
+
+# Regex Match (will be evaluated as truthy in a conditional statement if a match is found)
+"abcd".match(/b/) # => #<Regex::MatchData "b"> : (Regex::MatchData | Nil)
```

Comment:
I'd add example of getting all matches with `String#scan`: sth like `"Hello world".scan(/\w+/)`